### PR TITLE
Add a dashboard for configmap sizes

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/hold

### CODE REVIEW

File: cluster/ci/config/roles.yaml [hunk 1]
```diff
@@ -105,6 +105,32 @@ items:
     kind: ClusterRole
     apiGroup: rbac.authorization.k8s.io
     name: cluster-reader
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1beta1
+  metadata:
+    name: ci-admins-to-monitoring-admins
```
Comment (stevekuznetsov) [inline cluster/ci/config/roles.yaml]: @droslean this one

**Comment by stevekuznetsov** [comment]
@petr-muller @hongkailiu @droslean @bbguimaraes this is on staging now

File: cluster/ci/monitoring/mixins/grafana_dashboards/hook.jsonnet [hunk 2]
```diff
@@ -83,4 +83,22 @@ dashboard.new(
...
     y: 13,
   })
+.addPanel(
+    (graphPanel.new(
+        'configmap sizes',
+        description='prow_configmap_size_bytes / 1048576',
+        datasource='prometheus',
+        legend_alignAsTable=true,
+        legend_rightSide=true,
+        formatY1='percentunit',
```
Comment (hongkailiu) [inline cluster/ci/monitoring/mixins/grafana_dashboards/hook.jsonnet]: percentunit -> decbytes

File: cluster/ci/monitoring/mixins/grafana_dashboards/hook.jsonnet [hunk 3]
```diff
@@ -83,4 +83,22 @@ dashboard.new(
     x: 12,
     y: 13,
   })
+.addPanel(
+    (graphPanel.new(
+        'configmap sizes',
+        description='prow_configmap_size_bytes / 1048576',
```
Comment (hongkailiu) [inline cluster/ci/monitoring/mixins/grafana_dashboards/hook.jsonnet]: description='prow_configmap_size_bytes',

Comment (hongkailiu) [reply cluster/ci/monitoring/mixins/grafana_dashboards/hook.jsonnet]: Sorting if desired:
```
        legend_values=true,
        legend_current=true,
        legend_sort='current',
        legend_sortDesc=true,
```

File: cluster/ci/monitoring/mixins/grafana_dashboards/hook.jsonnet (hunk 2 above)
Comment (stevekuznetsov) [reply cluster/ci/monitoring/mixins/grafana_dashboards/hook.jsonnet]: I want this as a percentage, though

**Comment by stevekuznetsov** [comment]
Screenshot from 2019-07-11 13-49-39
working

**Pull request merged**